common/journal_entries/imperia_dead_man.txt

```diff
@@ -0,0 +1,109 @@
+﻿je_dead_man_main = {
+	icon = "gfx/interface/icons/event_icons/event_skull.dds"
+	group = je_group_historical_content
+	modifiers_while_active = {
+		dead_man_of_europe
+		country_cannot_enact_laws
+	}
+	immediate = {
+		set_variable = {
+			name = tur_leading_government_progress
+			value = 0
+		}
+		tur_set_dead_man_variables = yes
+	}
+	complete = {
+		scope:journal_entry = {
+			is_goal_complete = yes
+		}
+	}
+	on_complete = {
+		trigger_event = {
+			id = dead_man.2
+		}
+		tur_clear_dead_man_variables = yes
+	}
```

Comment:
![image](https://github.com/Imperia-Development-Victoria-3/Victoria-3-Dev/assets/44327039/aaeb4bbf-81e9-48a3-a51c-41dad287f3ec)
This is because the event options are dependent on the variables, which gets immediately cleared when the event pops. Perhaps clear the variables in an `after = {` section in the event?